routes/stats.tsx

```diff
@@ -44,89 +44,71 @@ export const handler: Handlers<StatsPageData, State> = {
+      borderColor: "#be185d",
     },
     {
-      title: "Users created",
-      values: props.data.usersCounts,
-      color: "#e85d04",
+      label: "Users created",
+      data: props.data.usersCounts,
+      borderColor: "#e85d04",
     },
     {
-      title: "Items created",
-      values: props.data.itemsCounts,
-      color: "#219ebc",
+      label: "Items created",
+      data: props.data.itemsCounts,
+      borderColor: "#219ebc",
     },
     {
-      title: "Votes",
-      values: props.data.votesCounts,
-      color: "#4338ca",
+      label: "Votes",
+      data: props.data.votesCounts,
+      borderColor: "#4338ca",
     },
   ];
 
-  const x = props.data.dates.map((date) =>
+  const labels = props.data.dates.map((date) =>
     new Date(date).toLocaleDateString("en-us", {
       month: "short",
       day: "numeric",
     })
   );
 
   return (
-    <main class="flex-1 p-4 grid gap-12 md:grid-cols-2">
-      {charts.map(({ color, title, values }) => {
-        const data = values.map((value) => Number(value));
-        const total = data.reduce(
-          (value, currentValue) => currentValue + value,
-          0,
-        );
-
-        return (
-          <div class="py-4">
-            <div class="text-center">
-              <h3>{title}</h3>
-              <p class="font-bold">{total}</p>
-            </div>
-            <Chart
-              container={{
-                class: "aspect-[2/1] mx-auto relative max-w-[100vw]",
-              }}
-              type="line"
-              options={{
-                plugins: {
-                  legend: { display: false },
-                },
-                interaction: {
-                  intersect: false,
-                },
-                scales: {
-                  x: {
-                    grid: { display: false },
-                  },
-                  y: {
-                    beginAtZero: true,
-                    grid: { display: false },
-                    max: Math.ceil(Math.max(...data) * 1.1),
-                    ticks: { stepSize: 1 },
-                  },
-                },
-              }}
-              data={{
-                labels: x,
-                datasets: [{
-                  label: title,
-                  data,
-                  borderColor: color,
-                  pointRadius: 0,
-                  cubicInterpolationMode: "monotone",
-                }],
-              }}
-            />
-          </div>
-        );
-      })}
+    <main class="flex-1 p-4 aspect-[2/1] mx-auto relative w-full">
+      <Chart
+        type="line"
+        options={{
+          plugins: {
+            title: {
+              display: true,
+              text: "Daily counts",
+            },
+          },
+          interaction: {
+            intersect: false,
+            mode: "index",
+          },
+          scales: {
+            x: {
+              grid: { display: false },
+            },
+            y: {
```

Comment:
**issue (non-blocking):** by removing `steps: { stepSize: 1 }` we might re-introduce some unwanted behavior as seen in https://github.com/denoland/saaskit/issues/276.  

*however* by having a joint chart containing all datasets it'd be less likely to occur in real life.
from a visual standpoint i'd also much prefer the current solution.

locally i was able to reproduce the behavior by simply clicking on one of the labels.  
there you can see that the y-axis is containing floats again:
![Bildschirm­foto 2023-07-05 um 08 30 31](https://github.com/denoland/saaskit/assets/8133892/05d17e13-9d83-4dab-a8bf-abeea30d53a7)

Reply:
`ticks: { precision: 0 }` might be a better, more suitable solution though.